avocado/plugins/jobs.py

```diff
@@ -153,7 +157,7 @@ def handle_show_command(self, config):
         # We could improve this soon with more data and colors
         self._print_job_details(data)
         LOG_UI.info("")
-        self._print_job_tests(results_data.get("tests"))
+        self._print_job_tests(results_data)
```

Comment:
The thing here is both `start` and `end` inside the `tests` are monotonic time. So I have to take the start local timestamp from one layer above.

Reply:
Yes, you are right here. I have looked more deeply into this problem and find out that we gather the `start` and `end` with `time.time()` in seconds, but we do not store it to the results file. I have opened PR #5768, which should help you with this when it is merged.

Reply:
Awesome, yeah that would be super helpful. Btw, I realize I should add the sign-off by "-s". I will fix them later.